### PR TITLE
test: Create unit tests for react components in packages/ui/components/top-banner/TopBanner

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## What does this PR do?

Create unit tests for TopBanner component

### CODE REVIEW

File: packages/ui/components/top-banner/topBanner.test.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  test("Should render a button", () => {
+    const mockOnClose = vi.fn();
+    render(<TopBanner text="the banner test" onClose={mockOnClose} />);
+
+    const btnElt = screen.getByRole("button");
+    expect(btnElt).toBeInTheDocument();
+
+    fireEvent.click(btnElt);
+    expect(mockOnClose).toHaveBeenCalled();
```
Comment (gitstart-calcom) [inline packages/ui/components/top-banner/topBanner.test.tsx:27]: Hello! As you can see there is a test failing and this is caused by the `onClose` prop not being called in the component, what can we do about this prop?

Comment (hariombalhara) [reply packages/ui/components/top-banner/topBanner.test.tsx:27]: Let's remove the functionality. I don't think anywhere we are passing onClose prop to it. Things are working through `actions` prop

**Review by hariombalhara — DISMISSED** [review]
Left a reply to work on.

**Comment by gitstart-calcom** [comment]
Hi @hariombalhara, the requested changes have been made. Please review

**Review by shivamklr — APPROVED** [review]
lgtm

**Pull request merged**